README.md

```diff
@@ -11,7 +11,7 @@ When the post is rendered on the front end, this same data is used to bootstrap
 ### Installation on WordPress.com VIP
 
 1. Place the plugin in a directory inside your theme, e.g.  `mytheme/inc/wordpress-simplechart`
-1. Follow the **Local app setup commands** steps below
+1. Follow the **Local app setup script** steps below
```

Comment:
yeah, that makes sense. we'd only need to include and version the compiled js and index.html